src/lib/polkadotApi.ts

```diff
@@ -66,9 +50,9 @@ export async function getBalances(apiPromise: ApiPromise, accountId: string): Pr
 }
 
 export async function getMsaInfo(apiPromise: ApiPromise, publicKey: string): Promise<MsaInfo> {
-  const received: u64 = (await apiPromise.query.msa.publicKeyToMsaId(publicKey)).unwrapOrDefault();
+  const received: u64 = (await apiPromise?.query.msa.publicKeyToMsaId(publicKey))?.unwrapOrDefault();
```

Comment:
Shouldn't need the `?`s in there, I think

Reply:
@JoeCap08055 I was getting "cannot on undefined" type of errors in the dom when I didn't have them.